### ISSUE TITLE
Set left/right sidebar width separately

### ISSUE THREAD

**Issue by Pil0tXia** [KosmosisDire/obsidian-webpage-export#442]
**Is your feature request related to a problem? Please describe.**
When editing `Layout Options - Sidebar Width`, the left and right sidebar will both use this width value. However, user may want to make the left sidebar wider and the right sidebar narrower.

**Describe the solution you'd like**
Add `Left Sidebar Width` and `Right Sidebar Width` options.

**Comment by KosmosisDire** [comment]
The sidebars are resizable by dragging on the edge of them. They keep their width, have you tried this? 

I do want to make it so you can set the default width of the sidebars independently though. Thanks for the suggestion

**Comment by Pil0tXia** [comment]
> The sidebars are resizable by dragging on the edge of them. They keep their width, have you tried this?

Yeah, dragging is easy. I'm hoping that those who are viewing the html page for the first time can dispense with the dragging step. Thanks~

**Comment by KosmosisDire** [comment]
Hey, I just released a beta version that should address this. If you could try it out and let me know if his issue is fixed and if you find any other issues. There are instructions for installing the beta in the readme :)
Thanks!

**Comment by Pil0tXia** [comment]
@KosmosisDire 

Hi, thank you for your development! Before I uninstalled 1.8.01, I installed 1.9.0-2b through BRAT and tried to run Local Website export after changing the width of the left and right sidebars. However, the plugin seemed to freeze and stayed on this page:

![QQ_1725119636507](https://github.com/user-attachments/assets/c163a26c-3e7d-4338-90a2-9f3de7249666)

This forced me to exit Obsidian. May you please help take a look?

**Comment by KosmosisDire** [comment]
How many files are in your vault?

**Comment by Pil0tXia** [comment]
@KosmosisDire 

About 100+ markdown files and 1000+ pictures. I encountered the white screen problem after I clicked the "Export" button of 1 markdown file with about 100 pictures in beta version.

**Comment by Pil0tXia** [comment]
@KosmosisDire 

I just uninstalled the release version and reinstalled the beta version and chose to export another note with less images and no AV1 encoded MP4 videos, then the plugin successfully exported the html file.

However, even though I set two different values for left and right sidebar width, the exported html page used the right sidebar width value for both left and right sidebars.

**Comment by Pil0tXia** [comment]
Translation to English:
The Local Website export method does not export the attachment folder along with it, while the Raw HTML Document export method exports unnecessary attachment folders.

The HTML files exported from the Local Website cannot display images correctly because they use relative links to reference images, which are not exported together.

My markdown notes use the following link to reference image and video files:
`![](assets/2024-05-13_我推的孩子/[Sakurato]%20Oshi%20no%20Ko%20[08v2][AVC-8bit%201080p%20AAC][CHS].mp4_001018.123.webp)`
Where the video file is encoded with AV1 and has an mp4 extension.

The Local Website export method output the following file:
```
ObsidianMDs.html // My vault name
```

The Raw HTML Document export method output the following files:
```
.
├── 2024-05-13_我推的孩子.html // My note file name
├── assets
│   └── 2024-05-13_我推的孩子 // My note file name
└── site-lib
    ├── metadata.json
    └── search-index.json

4 directories, 3 files
```

I hope this case can provide you with some insights.
